packages/webui/src/collector/CollectorClient.ts

```diff
@@ -70,6 +83,22 @@ export default class CollectorClient {
     });
   }
 
+  // set a timeout on http trace status to cleanup any traces
+  // where we don't receive a response in a timely manner
+  private _setHttpTimeout(id: string) {
+    setTimeout(() => {
```

Comment:
Is it worth recording the timeout against the trace id and cancelling it when we get a response?  Not personally sure whether much is gained other than a tick in the OCD box